src/components/structures/RoomView.tsx

```diff
@@ -1933,6 +1934,7 @@ export class RoomView extends React.Component<IRoomProps, IRoomState> {
                                 oobData={this.props.oobData}
                                 signUrl={this.props.threepidInvite?.signUrl}
                                 room={this.state.room}
```

Comment:
As I understand it, this line serves no purpose because we know that `this.state.room` is falsey.

I think we should remove the line while we're here, as it is confusing.

Reply:
good point, it is removed